src/cmd/policy.go

```diff
@@ -84,6 +98,39 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+
+	if err := ast.As(a.Value, &org); err != nil {
```

Comment:
My understanding is that if `a` is nil then `a.Value` would error but I'll look into this because I'm not sure

Reply:
hmm - ya this needs a test - its possible the rego machinery doesn't allow `a` to be nil because of the schema in but if it doesn't we'd get a panic on `a.Value` because accessing a nil struct field.

Reply:
tried these 2 tests and this is what I got, respectively:
`opslevel.repo.github()`
`opslevel.repo.github("","")`
```
$ go run main.go run policy -f github-repo.rego
Error: 1 error occurred: test.rego:9: rego_type_error: opslevel.repo.github: arity mismatch
	have: (???)
	want: (string, string)
exit status 1
$ go run main.go run policy -f github-repo.rego
5:26PM ERR 404: {"message":"Not Found","documentation_url":"https://docs.github.com/rest"}
```

Reply:
I'll have it error when it's an empty string just so that it doesn't make it to the github api call.

Reply:
Similar to what we did in the read file function, I'm assuming we'd want to log a message and exit the function call but continue the run execution in case there are other function calls that are correctly called?  Not sure how the app will expect bad function calls in a rego policy.

Reply:
The logs emitted by the CLI will probably be discoverable somehow - so whatever debugging information we can emit in the logs the better.

Reply:
Sounds good, just had it return error message, exit the function but proceed with the run.